### PR TITLE
Upating Container Runtime output on initialization without kubernetes

### LINKED ISSUE #13432: Show the container runtime when running without kubernetes
Currently we hide the container runtime output.

But there should be plenty of room to show it ?

No Kubernetes

```
😄  minikube v1.25.1 on Ubuntu 20.04
✨  Automatically selected the kvm2 driver. Other choices: virtualbox, ssh
👍  Starting minikube without Kubernetes minikube in cluster minikube
🔥  Creating kvm2 VM (CPUs=2, Memory=3900MB, Disk=20000MB) ...
🏄  Done! minikube is ready without Kubernetes!
╭───────────────────────────────────────────────────────────────────────────────────────╮
│                                                                                       │
│                       💡  Things to try without Kubernetes ...                        │
│                                                                                       │
│    - "minikube ssh" to SSH into minikube's node.                                      │
│    - "minikube docker-env" to point your docker-cli to the docker inside minikube.    │
│    - "minikube image" to build images without docker.                                 │
│                                                                                       │
╰───────────────────────────────────────────────────────────────────────────────────────╯
```

With Kubernetes

```
😄  minikube v1.25.1 on Ubuntu 20.04
✨  Automatically selected the kvm2 driver. Other choices: virtualbox, ssh
👍  Starting control plane node minikube in cluster minikube
🔥  Creating kvm2 VM (CPUs=2, Memory=3900MB, Disk=20000MB) ...
🐳  Preparing Kubernetes v1.23.1 on Docker 20.10.12 ...
    ▪ kubelet.housekeeping-interval=5m
    ▪ Generating certificates and keys ...
    ▪ Booting up control plane ...
    ▪ Configuring RBAC rules ...
🔎  Verifying Kubernetes components...
    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
🌟  Enabled addons: default-storageclass, storage-provisioner
🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
```

----

This sentence is somewhat weird: "Starting minikube without Kubernetes minikube"

~Probably better off as "Starting virtual machine minikube" (or "container", for KIC) ?~

EDIT: It did say "VM" on the line below, so maybe something else rhyming with node.

```
👍  Starting node minikube in cluster minikube
🔥  Creating kvm2 VM (CPUs=2, Memory=3900MB, Disk=20000MB) ...
```

And this one doesn't contain much info: "Done! minikube is ready without Kubernetes!"

~Slightly more useful as "Done! minikube is ready with Docker 20.10.12" instead~

EDIT: Possibly on a separate line, to show the fancy runtime emoji (like that whale)

```
🐳  Preparing Docker 20.10.12 ...
🏄  Done! minikube is ready without Kubernetes!
```

### PR DESCRIPTION
**Pull request by MdSahil-oss**
<h3>What type of PR is this ?</h3>
Add New Features.

<h3>What type of PR is this ?</h3>
fixes #13432 .

### CODE REVIEW

**Comment by MdSahil-oss** [comment]
@afbjorklund , Is this good or i need to improve it more.

**Comment by afbjorklund** [comment]
@MdSahil-oss : please rebase and squash (to remove the merge commits), and improve the commit message...

You cannot hardcode the container runtime, it has to come from the actual configuration (like _with_ Kubernetes)

File: pkg/minikube/node/start.go [hunk 1]
```diff
@@ -96,6 +96,7 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 		return nil, err
 	}
 	if stopk8s {
+		out.Step(style.Docker, "Preparing Docker 20.10.12 ...")
```
Comment (afbjorklund) [inline pkg/minikube/node/start.go:99]: This needs to be dynamic, and use the same code as when running _with_ Kubernetes

**Review by afbjorklund — CHANGES_REQUESTED** [review]

**Comment by MdSahil-oss** [comment]
@afbjorklund I made changes as per your suggestion

**Review by medyagh — COMMENTED** [review]
can you plz choose a more descriptive PR title and also paste the output of the minikube Before and After this PR in both normal cases and without kubernets ?

**Comment by spowelljr** [comment]
@MdSahil-oss could you post the output in the PR description like @medyagh mentioned.

Just run minikube on master branch and copy and paste the output then do the same with this branch, something like:
**Before:**
```
😄  minikube v1.25.1 on Ubuntu 20.04
✨  Automatically selected the kvm2 driver. Other choices: virtualbox, ssh
👍  Starting minikube without Kubernetes minikube in cluster minikube
...
```

**After:**
```
😄  minikube v1.25.1 on Ubuntu 20.04
✨  Automatically selected the kvm2 driver. Other choices: virtualbox, ssh
👍  Starting minikube without Kubernetes minikube in cluster minikube
...
```
That way we can easily see the UI changes without having to run it, thanks!

**Comment by MdSahil-oss** [comment]
@spowelljr sure, I would have done it but i'm getting an error on my ubuntu machine related to lint while running minikube.

**Comment by MdSahil-oss** [comment]
@klaases I'm still getting error on my ubuntu machine related to lint while running minikube.

**Pull request closed without merging**